### DISCUSSION TITLE
How do you use DISCORD_NOTIFY_GROUP_TEST

### DISCUSSION THREAD

**Discussion by FloatingBlob** [jef/streetmerchant#2652] [Help] [answered]
I feel really dumb asking this but for some reason nothing happens.

**Comment by hofq** [comment, marked as answer]
The Role you enter should be getting an Notification if you run `npm run test:notification` on the Host/ in the Container.

**Reply by hofq** [reply]
In Docker for me only worked `test:notification:production`

**Reply by FloatingBlob** [reply]
thank you!
